src/main/java/org/opentripplanner/api/model/ApiAlert.java

```diff
@@ -8,4 +8,5 @@
     public String alertUrl;
     /** null means unknown */
     public Date effectiveStartDate;
+    public Date effectiveEndDate;
```

Comment:
Should this be mentioned in the migration guide?

Reply:
Added